tests/bsim/bluetooth/audio/src/common.c

```diff
@@ -152,17 +152,22 @@ BT_CONN_CB_DEFINE(conn_callbacks) = {
 
 void setup_broadcast_adv(struct bt_le_ext_adv **adv)
 {
+	struct bt_le_per_adv_param per_adv_param = *BT_BAP_PER_ADV_PARAM_BROADCAST_SLOW;
 	struct bt_le_adv_param ext_adv_param = *BT_BAP_ADV_PARAM_BROADCAST_SLOW;
+	int err;
 
 	/* Zephyr Controller works best while Extended Advertising interval is a multiple
-	 * of the ISO Interval minus 10 ms (max. advertising random delay). This is
-	 * required to place the AUX_ADV_IND PDUs in a non-overlapping interval with the
-	 * Broadcast ISO radio events.
+	 * of the Broadcast ISO Interval minus 10 ms (max. advertising random delay).
+	 *
+	 * Zephyr Controller also needs the Periodic Advertising Interval to be multiple of the
+	 * Broadcast ISO interval.
+	 *
+	 * To cover most high reliability tests we need a Periodic Advertising Interval of 150 ms.
 	 */
-	ext_adv_param.interval_min -= BT_GAP_MS_TO_ADV_INTERVAL(10U);
-	ext_adv_param.interval_max -= BT_GAP_MS_TO_ADV_INTERVAL(10U);
-
-	int err;
+	ext_adv_param.interval_min = BT_GAP_MS_TO_ADV_INTERVAL(140U);
+	ext_adv_param.interval_max = BT_GAP_MS_TO_ADV_INTERVAL(140U);
+	per_adv_param.interval_min = BT_GAP_MS_TO_PER_ADV_INTERVAL(150U);
+	per_adv_param.interval_max = BT_GAP_MS_TO_PER_ADV_INTERVAL(150U);
```

Comment:
We should not define a set of suggested values and then not use them ourselves. Suggest to modify the macros to use 150ms instead. The advertising interval values are not defined by the BAP spec